code/game/machinery/dish_drive.dm

```diff
@@ -26,6 +26,7 @@
 	var/time_since_dishes = 0
 	var/suction_enabled = TRUE
 	var/transmit_enabled = TRUE
+	var/list/dish_drive_contents = list()
```

Comment:
if you're adding this, no reason not to lazylist it

Reply:
Wasn't sure if it was better to just lazyinitlist it, or lazylen/add where applicable.

Looking at existing code, the latter method seemed to be preferred?